src/scripts/site.js

```diff
@@ -620,514 +589,6 @@ var AppInfo = {};
-            roles: "admin"
-        });
-        defineRoute({
-            path: "/usernew.html",
-            dependencies: [],
-            autoFocus: false,
-            roles: "admin"
-        });
-        defineRoute({
-            path: "/userparentalcontrol.html",
-            dependencies: [],
-            autoFocus: false,
-            roles: "admin"
-        });
-        defineRoute({
-            path: "/userpassword.html",
-            dependencies: ["emby-input", "emby-button", "emby-checkbox"],
-            autoFocus: false,
-            controller: "scripts/userpasswordpage"
-        });
-        defineRoute({
-            path: "/userprofiles.html",
-            dependencies: [],
-            autoFocus: false,
-            roles: "admin"
-        });
-        defineRoute({
-            path: "/wizardremoteaccess.html",
-            dependencies: ["dashboardcss"],
-            autoFocus: false,
-            anonymous: true,
-            controller: "dashboard/wizardremoteaccess"
-        });
-        defineRoute({
-            path: "/wizardfinish.html",
-            dependencies: ["emby-button", "dashboardcss"],
-            autoFocus: false,
-            anonymous: true,
-            controller: "dashboard/wizardfinishpage"
-        });
-        defineRoute({
-            path: "/wizardlibrary.html",
-            dependencies: ["dashboardcss"],
-            autoFocus: false,
-            anonymous: true
-        });
-        defineRoute({
-            path: "/wizardsettings.html",
-            dependencies: ["dashboardcss"],
-            autoFocus: false,
-            anonymous: true,
-            controller: "dashboard/wizardsettings"
-        });
-        defineRoute({
-            path: "/wizardstart.html",
-            dependencies: ["dashboardcss"],
-            autoFocus: false,
-            anonymous: true,
-            controller: "dashboard/wizardstart"
-        });
-        defineRoute({
-            path: "/wizarduser.html",
-            dependencies: ["dashboardcss", "emby-input"],
-            controller: "scripts/wizarduserpage",
-            autoFocus: false,
-            anonymous: true
-        });
-        defineRoute({
-            path: "/videoosd.html",
-            dependencies: [],
-            transition: "fade",
-            controller: "scripts/videoosd",
-            autoFocus: false,
-            type: "video-osd",
-            supportsThemeMedia: true,
-            fullscreen: true,
-            enableMediaControl: false
-        });
-        defineRoute({
-            path: "/configurationpage",
-            dependencies: [],
-            autoFocus: false,
-            enableCache: false,
-            enableContentQueryString: true,
-            roles: "admin"
-        });
-        defineRoute({
-            path: "/",
-            isDefaultRoute: true,
-            autoFocus: false,
-            dependencies: []
-        });
-    }
-
-    function getPluginPageContentPath() {
```

Comment:
why remove?

Reply:
probably a function that's not being used anywhere. I confirmed this with the other PR that don't have these changes.